doc/sphinx/source/howto/import.rst

```diff
@@ -57,14 +57,16 @@ skip ahead to the next section, :ref:`install_ngonnx`.
       $ cd build && cmake ../ -DNGRAPH_USE_PREBUILT_LLVM=TRUE
       $ make install
 
-#. Build the Python package (binary wheel) for ngraph and set up an env for ONNX:
+#. Build the Python package (binary wheel) for ngraph and set up an env for ONNX;
+   be sure to export the ``NGRAPH_CPP_BUILD_PATH`` where the ``ngraph_dist`` was 
+   installed. 
 
    .. code-block:: console
 
       $ cd ngraph/python
       $ git clone --recursive -b allow-nonconstructible-holders https://github.com/jagerman/pybind11.git
       $ export PYBIND_HEADERS_PATH=$PWD/pybind11
-      $ export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist
+      $ export NGRAPH_CPP_BUILD_PATH=path/to/ngraph_dist
```

Comment:
And now `NGRAPH_CPP_BUILD_PATH` joins the fray. It's almost like we lack a system architect to keep everything consistent. Now we have three environment variables pointing to the same thing.